gravitee-apim-rest-api/gravitee-apim-rest-api-service/src/main/java/io/gravitee/rest/api/service/v4/PlanService.java

```diff
@@ -31,21 +31,17 @@
  * @author GraviteeSource Team
  */
 public interface PlanService {
-    PlanEntity findById(final ExecutionContext executionContext, final String plan);
+    PlanEntity findById(ExecutionContext executionContext, String plan);
```

Comment:
Why did you keep this method here ?

Reply:
to avoid to cast to PlanEntity when I actually need a planentity same thing with Set<PlanEntity> findByApi.